src/framework/ui/checkbox/checkbox.component.tsx

```diff
@@ -54,34 +59,46 @@ export class CheckBox extends React.Component<Props> {
     }
   };
 
+  private getSelectSize = (indeterminate: boolean, width: number, height: number): StyleType => {
+    if (indeterminate) {
+      return {
+        width: width - width * 0.18,
+        height: height * 0.18,
+      };
+    } else {
+      return {
+        width: width * 0.5,
+        height: height * 0.5,
+      };
+    }
+  };
+
   private getComponentStyle = (style: StyleType): StyleType => {
     const {
       textColor,
       textFontSize,
       textFontWeight,
-      textMarginLeft,
-      selectWidth,
-      selectHeight,
-      selectBackgroundColor,
+      textLineHeight,
+      checkMarkColor,
       highlightWidth,
       highlightHeight,
       highlightBorderRadius,
       highlightBackgroundColor,
       ...containerParameters
     } = style;
+    const { indeterminate } = this.props;
 
     return {
       selectContainer: containerParameters,
       text: {
         color: textColor,
         fontSize: textFontSize,
         fontWeight: textFontWeight,
-        marginLeft: textMarginLeft,
+        lineHeight: textLineHeight,
       },
       select: {
```

Comment:
prefer renaming `select` to `icon` due to `renderIcon` function name

Reply:
Done